app/models/claim/advocate_interim_claim.rb

```diff
@@ -0,0 +1,5 @@
+module Claim
```

Comment:
📓  Literally a place holder, for now, since there isn't yet any logic associated with it apart from being able to select it in the "claim type selection page"